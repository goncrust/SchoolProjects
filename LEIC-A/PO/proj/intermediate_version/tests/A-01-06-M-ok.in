1
wqrqwrwqqw
0
