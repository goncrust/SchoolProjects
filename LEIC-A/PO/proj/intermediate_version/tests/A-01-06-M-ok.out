MENU PRINCIPAL
1 - Abrir
2 - Guardar
3 - Gestão de clientes
4 - Gestão de terminais
5 - Menu de consultas
6 - Ver saldo global
0 - Sair
Escolha uma opção: Ficheiro a abrir: Abrir: Operação inválida: Problema ao abrir ficheiro: Erro a processar ficheiro wqrqwrwqqw
MENU PRINCIPAL
1 - Abrir
2 - Guardar
3 - Gestão de clientes
4 - Gestão de terminais
5 - Menu de consultas
6 - Ver saldo global
0 - Sair
Escolha uma opção: 